mne/preprocessing/maxwell.py

```diff
@@ -554,7 +490,7 @@ def _update_info(raw, origin, int_order, ext_order, nsens, nmoments):
     max_info_dict = dict(max_st={}, sss_cal={}, sss_ctc={},
                          sss_info=sss_info_dict)
 
-    block_id = dict(machid=-1 * np.ones(2), secs=-1, usecs=-1, version=-1)
+    block_id = dict(machid=get_machid(), secs=-1, usecs=-1, version=-1)
     proc_block = dict(max_info=max_info_dict, block_id=block_id,
                       creator='mne-python v%s' % __version__,
                       date=-1, experimentor='')
```

Comment:
If you can't figure out what they mean, you can also look in the MNE manual, and the MaxFilter manual. I think the MaxFilter one especially has an outline of what all the fields mean.